website/docs/customize/blueprints/v1/tags.mdx

```diff
@@ -79,6 +80,24 @@ configure_flow:
 Looks up any model and resolves to the the matches' primary key.
 First argument is the model to be queried, remaining arguments are expected to be pairs of key=value pairs to query for.
 
+#### `!FindMany`
+
+Examples:
+
+{/* prettier-ignore-start */}
+
+```yaml
+configure_flow: !Find [authentik_flows.flow, [slug, default-password-change]]
+```
```

Comment:
I'm guessing this is a leftover that was supposed to be removed?

Reply:
Yep, good catch.